enos/modules/get_local_metadata/scripts/get-local-version.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
```

Comment:
`/usr/bin/env bash` did not work for me on macOS. I suppose it could be a local configuration issue but is it not reasonable to assume `/bin/bash` will exist?